### DISCUSSION TITLE
What version of Node.js do you use?

### DISCUSSION THREAD

**Discussion by boneskull** [appium/appium#17514] [General]
Appium v2 currently claims support for **Node.js v14.0.0 and newer**.

The project is considering limiting the support for non-"active" Node.js LTS releases--those in "maintenance" mode--to the _latest minor version_  (see [nodejs/release](/nodejs/release) for more information).

As of this writing, the latest minor version of the maintenance LTS line is **v14.20.0**.  That means Appium would support Node.js v14.20.0, v16.x and v18.x.

My question for the community is:

1. What version of Node.js do you use to run Appium v2?
2. If you use Node.js v14.0.0-v14.19.x, would it be problematic to upgrade?

_Note: the Node.js project recommends keeping current with the latest maintenance LTS releases, as these will contain critical security patches._

_Another note: Appium 1.x would be unaffected by any policy change here._

**Comment by RSM-SBA** [comment]
node v16.16 with appium v2
